src/server/logging/log_format.js

```diff
@@ -111,30 +111,35 @@ export default class TransformObjStream extends Stream.Transform {
         'load'
       ]));
       data.message  = ansicolors.brightBlack('memory: ');
-      data.message += numeral(data.proc.mem.heapUsed).format('0.0b');
+      data.message += numeral(_.get(data, 'proc.mem.heapUsed')).format('0.0b');
       data.message += ' ';
       data.message += ansicolors.brightBlack('uptime: ');
-      data.message += numeral(data.proc.uptime).format('00:00:00');
+      data.message += numeral(_.get(data, 'proc.uptime')).format('00:00:00');
       data.message += ' ';
       data.message += ansicolors.brightBlack('load: [');
-      data.message += data.os.load.map(function (val) {
+      data.message += _.get(data, 'os.load', []).map(function (val) {
         return numeral(val).format('0.00');
       }).join(' ');
       data.message += ansicolors.brightBlack(']');
       data.message += ' ';
       data.message += ansicolors.brightBlack('delay: ');
-      data.message += numeral(data.proc.delay).format('0.000');
+      data.message += numeral(_.get(data, 'proc.delay')).format('0.000');
     }
     else if (data.type === 'error') {
       data.level = 'error';
-      data.message = event.error.message;
       data.error = serializeError(event.error);
       data.url = event.url;
+
+      const message = _.get(event, 'error.message');
+      data.message = (message && message !== '') ? message : JSON.stringify(event);
     }
     else if (event.data instanceof Error) {
+      data.type = 'error';
       data.level = _.contains(event.tags, 'fatal') ? 'fatal' : 'error';
-      data.message = event.data.message;
       data.error = serializeError(event.data);
+
+      const message = _.get(event, 'data.message');
+      data.message = (message && message !== '') ? message : JSON.stringify(event);
```

Comment:
I'm iffy on stringifying the entire event in case it contains sensitive information - we'd have to audit usage of this to really make sure it's okay.  

Thoughts on using a default empty string instead?

Reply:
Good catch. Empty string would be better. Or what about a static string like `The error to be logged contained no message!`

Reply:
👍  the static string, it'll be easier for processing.  My only concern with the longer message is if it gets messaged frequently.   - seems to be relatively common in logs, thoughts on that?

Reply:
spoke with @tsullivan offline, this case shouldn't happen so we want the error to be more verbose than -